impl/d3d11_manager.cpp

```diff
@@ -735,8 +735,8 @@ void d3d11_manager::update_screen_size(const position& screen_size)
 	_wvp[0] = 2.f / _screen_size.x;
 	_wvp[5] = 2.f / -_screen_size.y;
 	_wvp[10] = 0.5f;
-	_wvp[12] = (_screen_size.x + 1.f) / -_screen_size.x;
-	_wvp[13] = (_screen_size.y + 1.f) / _screen_size.y;
+	_wvp[12] = (_screen_size.x) / -_screen_size.x;
```

Comment:
Can you replace this and the following line by -1,1 respectively since the computation result is constant.

Reply:
Good catch, fixed